src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -63,3 +73,32 @@ test('testing button text updates with default ad params', () => {
     expect(button.innerText).toBe('Find Nearest Location');
 });
 
+test('Map constructor in Maps API is accessed on button click', () => {
+    const eventData = {
+        args: {
+            creativeData: {
+                adParameters: '{"buttonLabel": "Place"}',
```

Comment:
no need to specify buttonLabel here if it's not being used for the test

Reply:
got it!